src/settings/ManageAuthoritySourceFiles/getValidators.js

```diff
@@ -50,7 +53,11 @@ const validators = {
 
     return undefined;
   },
-  [authorityFilesColumns.START_NUMBER]: function validateStartNumber({ hridManagement }) {
+  [authorityFilesColumns.START_NUMBER]: function validateStartNumber({ hridManagement, source }) {
+    if (source === SOURCES.FOLIO) {
+      return undefined;
+    }

```

Comment:
Maybe we should do the same for the other fields.

Reply:
Other fields in FOLIO records are filled by default, so I think it's ok